phpstan-baseline.neon

```diff
@@ -0,0 +1,66 @@
+parameters:
+	ignoreErrors:
+		-
+			message: "#^Access to constant SPAN_KIND_INTERNAL on an unknown class Opentelemetry\\\\Proto\\\\Trace\\\\V1\\\\Span\\\\SpanKind\\.$#"
+			count: 1
+			path: src/Instrumentation/CommandInstrumentation.php
+
+		-
+			message: "#^Access to constant STATUS_CODE_ERROR on an unknown class Opentelemetry\\\\Proto\\\\Trace\\\\V1\\\\Status\\\\StatusCode\\.$#"
+			count: 1
+			path: src/Instrumentation/CommandInstrumentation.php
+
+		-
+			message: "#^Method Shopware\\\\OpenTelemetry\\\\Instrumentation\\\\CommandInstrumentation\\:\\:register\\(\\) has no return type specified\\.$#"
```

Comment:
would be cool, if you could easy stuff like no type for parameter or return values

Reply:
Fixed most of them